### PR TITLE
fix(dashboard): handle deleted products/variants in the order domain

### PR DESCRIPTION
**Pull request by fPolic**
**What**
- safely handle deleted product/variant of an item in the order detail flows

---

FIXES CMRC-649
FIXES CMRC-656

### CODE REVIEW

File: packages/admin/dashboard/src/routes/orders/order-create-claim/components/add-claim-items-table/use-claim-item-table-columns.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ export const useClaimItemTableColumns = (currencyCode: string) => {
         id: "product",
         header: () => <ProductHeader />,
         cell: ({ row }) => (
-          <ProductCell product={row.original.variant.product} />
+          <ProductCell product={row.original.variant?.product} />
```
Comment (srindom) [inline packages/admin/dashboard/src/routes/orders/order-create-claim/components/add-claim-items-table/use-claim-item-table-columns.tsx]: thought: can this be decoupled so it only relies on the line item data?

**Review by carlos-r-l-rodrigues — APPROVED** [review]

**Review by srindom — APPROVED** [review]
LGTM w. comment

**Comment by olivermrbl** [comment]
@fPolic, can we merge this?

**Comment by fPolic** [comment]
@olivermrbl, yes 🙏

**Pull request merged**